syft/pkg/type.go

```diff
@@ -51,6 +53,8 @@ func (t Type) PackageURLType() string {
 		return packageurl.TypeRPM
 	case GoModulePkg:
 		return packageurl.TypeGolang
+	case RustPkg:
+		return "cargo"
```

Comment:
I did create https://github.com/package-url/packageurl-go/pull/24 to get a type for rust added, but judging by the activity on that repo, it seems unlikely to be merged anytime soon

Reply:
yup yup, we ran into the same thing 👍